Automattic/wp-calypso
# [E2E][AT][GB] Add new test criteria for AT GB edge nightly E2E testing

## Proposed Changes

Follow-up to: https://github.com/Automattic/wp-calypso/pull/77639 and https://github.com/Automattic/wp-calypso/pull/77711

* Add a new test criteria that will be selected when a `GUTENBERG_NIGHTLY` env var is set, which will cause the `gutenberg` version to be set to `nightly` and the right `nightly` account/site to be selected for the test to run against.

## Testing Instructions

* Unit tests should pass.
* E2E tests should pass.
* Test with the GB AT nighly build (Desktop suffices), make sure no tests fail because of unexpected JS errors or account not found errors. Result is [here](https://teamcity.a8c.com/buildConfiguration/calypso_WPComTests_gutenberg_atomic_nightly_desktop/10183210?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true). As long as there are no "account not found" errors, it should be considered 🟢
